code/game/objects/structures/transit_tubes/transit_tube.dm

```diff
@@ -11,7 +11,7 @@
 	pass_flags_self = PASSGLASS
 	var/tube_construction = /obj/structure/c_transit_tube
 	var/list/tube_dirs //list of directions this tube section can connect to.
-	var/exit_delay = 1
+	var/exit_delay = 0
```

Comment:
Setting it to zero is essentially equivalent to teleportation, at least when I tried it.